app/templates/macros/page_macros.html

```diff
@@ -2,30 +2,37 @@
     <div id="editor-{{ editable_html_obj.editor_name }}" contenteditable="false">
         {{ editable_html_obj.value|safe }}
     </div>
-
     {% if current_user.is_admin() %}
-       <button class="ui button start-edit">
+        <button class="ui button start-edit">
         Edit
-       </button>
-       <button class="ui primary button end-edit">
+        </button>
+        <p class= "updated_alert">Updated successfully</p>
+        <button class="ui primary button end-edit">
         Save
-       </button>
+        </button>
+        <button class="ui primary button cancel-edit">
+        Cancel
+        </button>
     {% endif %}
-
     <script>
     var editorIDName = "editor-{{ editable_html_obj.editor_name }}";
+    var original_data;
     $(document).ready(function() {
         {% if current_user.is_admin() %}
             $(".end-edit").hide();
+            $(".cancel-edit").hide();
+            $(".updated_alert").hide();
         {% endif %}
         $(".start-edit").click(function() {
             CKEDITOR.disableAutoInline = true;
-            var editor = CKEDITOR.inline(editorIDName, {
+            var editor = CKEDITOR.replace(editorIDName, {
```

Comment:
I changed the inline to replace to make the text box bigger. If I had left it as inline then the text box would have still been the same shape. I guess it was just  a semantic UI issue for that happening. I also did this part with rani because it was causing me problems. So you can ask her about it as well if you want.